app/views/users/edit.html.slim

```diff
@@ -107,7 +107,12 @@
   p.help-block This information will only be visible to yourself and the organizers.
   = f.input :tshirt_size, as: :select, collection: User::TSHIRT_SIZES.map { |k| [k, k] }, label: 'T-Shirt size', blank: false, required: false, hint: 'For sponsor T-Shirts, in case they send some.'
   = f.input :tshirt_cut, as: :select, collection: User::TSHIRT_CUTS.map { |s| [s, s] }, label: 'T-Shirt cut', include_blank: true
-  = f.input :postal_address, hint: "Please give your postal address, including your full name, so we can send things we've received from our sponsors for you :)"
+
+  h4 Shipping Address
+  = f.simple_fields_for :postal_addresses do |pa|
+    = pa.input :street
+    = pa.input :state
+    = pa.input :zip, required: true, hint: "Please give your postal address, so we can send things we've received from our sponsors for you :)"
```

Comment:
See my comments above about making use of `nested_attributes` here.
Aside that: why making only one of the fields required? And what about users who don't want to and don't need to provide an address? We only require the address from students, other users of the app are not required to provide this information. With making the field required like it is now, no user is able to update their profile without adding an address with a zipcode 😉 

Also: the hint does not make too much sense for the `zip` field alone. This looks more like a general hint we'd put above or below the address fields. I'd therefore remove it from the individual field and put it on the address fields in general.